packages/@aws-cdk/aws-eks/lib/cluster-resource.ts

```diff
@@ -141,6 +141,7 @@ export class ClusterResource extends CoreConstruct {
       actions: [
         'ec2:DescribeSubnets',
         'ec2:DescribeRouteTables',
+        'ec2:DescribeVpcs'
```

Comment:
The thing is that this already exists here:

https://github.com/aws/aws-cdk/blob/fe6ec39f92819d451e11df79e3d6145868818da1/packages/%40aws-cdk/aws-eks/lib/cluster-resource.ts#L179-L186

However, this only grants it for the specific VPC its going to use. Why do we need permissions to describe **ALL** VPC's? 

Also, did you add this because you hit the error described [here](https://github.com/aws/aws-cdk/issues/9027)? Im not sure this explains the sporadic behavior.